### ISSUE TITLE
AssertJ-Guava not compatible with AssertJ-Core 3.9.1

### ISSUE THREAD

**Issue by PascalSchumacher** [joel-costigliola/assertj-guava#39]
When I update https://github.com/joel-costigliola/assertj-examples to use AssertJ-Core 3.9.1 this failure occurs:

```
java.lang.NoSuchMethodError: org.assertj.core.api.Assertions.assertThat(Ljava/lang/Object;)Lorg/assertj/core/api/AbstractObjectAssert;
	at org.assertj.examples.guava.OptionalAssertionsExamples.optional_assertions_examples(OptionalAssertionsExamples.java:34)
```

I believe the cause is the binary incompatibility reported in https://github.com/joel-costigliola/assertj-core/issues/1190 and https://github.com/joel-costigliola/assertj-core/issues/1194

**Comment by joel-costigliola** [comment]
we need to release a new assertj-guava indeed

**Comment by joel-costigliola** [comment]
Fixed in e40f8844419531795f99eb3368bd3689fb9444eb
